lib/ansible/modules/network/ios/ios_command.py

```diff
@@ -198,7 +198,7 @@ def main():
     match = module.params['match']
 
     while retries > 0:
-        responses = run_commands(module, commands)
+        responses, timestamps = run_commands(module, commands, return_timestamps=True)
```

Comment:
Why go through all this trouble instead of

```
start = get_timestamp()
responses = run_commands(module, commands)
end = get_timestamp()
timestamps = (start, end)
```

Reply:
Ah, yes, I see that now